servers/rendering/renderer_rd/forward_clustered/scene_shader_forward_clustered.cpp

```diff
@@ -656,6 +657,11 @@ void SceneShaderForwardClustered::init(RendererStorageRD *p_storage, const Strin
 		actions.renames["CUSTOM2"] = "custom2_attrib";
 		actions.renames["CUSTOM3"] = "custom3_attrib";
 
+		// not implemented but need these just in case code is in the shaders
+		actions.renames["VIEW_INDEX"] = "0";
+		actions.renames["VIEW_MONO_LEFT"] = "0";
```

Comment:
I would kinda expect that VIEW_INDEX 0 and 1 should be enough. Also should it not be MULTIVIEW_INDEX ? I am not sure this is very useful outside multiview.

Reply:
`VIEW_INDEX` is the actual variable, it's only 0 here because its turned off in the forward clustered, this code is only there so the compiler doesn't throw up. See the mobile implementation for the proper code.